packages/shared/src/contexts/FeedContext.tsx

```diff
@@ -112,9 +107,13 @@ const FeedContext = React.createContext<FeedContextData>(
 export function FeedLayoutProvider({
   children,
 }: PropsWithChildren): ReactElement {
-  const { sidebarExpanded } = useContext(SettingsContext);
+  const { sidebarExpanded } = useSettingsContext();
   const { sidebarRendered } = useSidebarRendered();
-  const feedPageSizes = useFeature(feature.feedPageSizes);
+  const { isPlus } = usePlusSubscription();
+  const feedPageSizes = useConditionalFeature({
+    feature: feature.feedPageSizes,
+    shouldEvaluate: !isPlus,
```

Comment:
Can't remember how I build it, but this will also return default if you where previously enrolled and now plus member right?

If so it's fine, just want to avoid you enrolling -> getting plus -> still seeing the experiment

Reply:
Yeah, when `shouldEvaluate` evaluates to false, it fallbacks to the default value